src/main/scala/cosas/ops/types/SerializeDenotations.scala

```diff
@@ -0,0 +1,44 @@
+package ohnosequences.cosas.ops.types
+
+import ohnosequences.cosas._, types._, typeSets._, fns._
+
+// TODO msg not found
+trait SerializeDenotations[Denotations <: AnyTypeSet, V]
+extends Fn2[Denotations, Map[String,V]] with
```

Comment:
Why does it take a map as input? I think it should take only the typeset and if it's empty, then it's an empty Map,